src/bench/pqs/linden.cpp

```diff
@@ -35,16 +36,38 @@ Linden::~Linden()
 
 void
 Linden::insert(const uint32_t &key,
-               const uint32_t & /* Unused */)
+               const uint32_t &value)
 {
-    linden_insert(m_q->pq, key);
+    /*Add 1 to key to support 0 keys*/
+    linden_insert(m_q->pq, key+1, value);
+}
+
+void
+Linden::insert(const size_t &key,
```

Comment:
Nit: Is this overload really needed? Same for all other sites. Can't we instead make sure that call-sites all use uint32_t's?